libs/tdigest/src/test/java/org/elasticsearch/tdigest/TDigestTests.java

```diff
@@ -152,7 +152,7 @@ public void testQuantile() {
         hist2.compress();
         double x1 = hist1.quantile(0.5);
         double x2 = hist2.quantile(0.5);
-        assertEquals(Dist.quantile(0.5, data), x1, 0.2);
+        assertEquals(Dist.quantile(0.5, data), x1, 0.25);
```

Comment:
This is orthogonal, noticed it in CI: https://gradle-enterprise.elastic.co/s/36zxzfevfwrcc

Verified that it happens regardless of the change in `MergingDigest`.